.github/workflows/generated-linux-binary-manywheel-nightly.yml

```diff
@@ -1465,6 +1577,118 @@ jobs:
       conda-pytorchbot-token-test: ${{ secrets.CONDA_PYTORCHBOT_TOKEN_TEST }}
     uses: ./.github/workflows/_binary-upload.yml
 
+  manywheel-py3_9-xpu-build:
```

Comment:
Most of the code is copy-past. Compared with Py3.8, is the python version the only difference? I'm just wondering if we can reuse the code (https://yaml.org/spec/1.2.2/#rule-c-ns-anchor-property)

Reply:
Hi @EikanWang , this workflow is generated by script `.github/scripts/generate_ci_workflows.py` based on template [.github/templates/linux_binary_build_workflow.yml.j2](https://github.com/pytorch/pytorch/blob/4da45c5243a7be402723660c4e58cf0249411323/.github/templates/linux_binary_build_workflow.yml.j2)